src/components/SelectBox.js

```diff
@@ -43,16 +44,30 @@ class SelectBox extends React.Component {
       this.props.onChange(id)
     }
   }
+  getValidationState () {
+    const { validationState } = this.props
```

Comment:
minor one: consider passing validation state as parameter. This would make `getValidationState` a pure function.
It brings some nice features (although we don't use them now).

Reply:
It won't be fully pure function, because we are using external resource, so IMO it's not matter,WDYT?